cmake/patches/composable_kernel/Fix_Clang_Build.patch

```diff
@@ -198,3 +198,41 @@ index c035e7e56..8c5f36d2e 100644
          set_target_properties(${INSTANCE_NAME} PROPERTIES POSITION_INDEPENDENT_CODE ON)
          clang_tidy_check(${INSTANCE_NAME})
          set(result 0)
+--- ./include/ck/utility/amd_buffer_addressing.hpp      2024-09-05 10:12:33.343091000 +0800
++++ ./include/ck/utility/amd_buffer_addressing_new.hpp  2024-09-05 10:12:20.276686000 +0800
```

Comment:
@hann-wang, is it expected that the filename is changed here? I think CK is still use the original filename:
https://github.com/ROCm/composable_kernel/blob/f16ebf82d42d49dc8905e5d6ac66eee8b25cd524/include/ck_tile/core.hpp#L9